packages/web-app-files/src/helpers/resources.js

```diff
@@ -154,7 +169,11 @@ export function buildSpace(space) {
     ownerId: '',
     disabled,
     spaceQuota: space.quota,
-    spacePermissions,
+    spaceRoles: {
```

Comment:
This introduces knowledge about the available space sharing roles by hardcoding their names here. You have everything at hand to build this `spaceRoles` map from looping over the result of `SpacePeopleShareRoles.list()`

Reply:
👍 

Not a 100% sure if I did it the most convenient way though...